components/search/Search.tsx

```diff
@@ -7,35 +7,35 @@ import { InstantSearch, connectSearchBox } from "react-instantsearch-dom";
 import Content from "./content";
```

Comment:
Remove all packages/variables that are not used. Also install any dependancies that you used with the -s flag so its in the package.json

Reply:
i was under the impression that you didn't need to install with -s after npm 5?

Reply:
<img width="589" alt="Screen Shot 2020-05-31 at 4 48 47 PM" src="https://user-images.githubusercontent.com/35946905/83365660-a04ee600-a35e-11ea-8386-7fff4e3b85c2.png">

When I try `npm run build`, I get the above error. Is there a package that I am missing?

Reply:
right, i think its a problem with using the lite version with typescript, as per https://github.com/algolia/algoliasearch-client-javascript/issues/666, it should work with the full version

Reply:
Can you add the full version and delete the lite import?

Reply:
it should already be in the latest push

Reply:
Okie i see it

<img width="348" alt="Screen Shot 2020-05-31 at 6 46 38 PM" src="https://user-images.githubusercontent.com/35946905/83368853-27a45580-a36f-11ea-9ae5-8fd4d6ffd883.png">
When the hamburger is clicked, can you make sure that this is shows up when clicked? You should use the ant design popover component, https://ant.design/components/popover/ for this. Right now the user has now way logging in. If the user is not logged in show the hamburger else show their profile image which should lead to their profile

Reply:
the popover shows up and the links are clickable, is there somewhere i can grab the picture? i dont think i can get it straight from the figma and it's not in images